src/components/scss/Profile.scss

```diff
@@ -1,7 +1,74 @@
 .Profile__block {
-  background-color: $gray;
-  padding: 16px;
+  background-color: $white;
+  padding: 32px;
   display: flex;
   flex-direction: column;
   flex-basis: 30%;
+  border-radius: 25px;
+}
+
+.Profile__personalInfoBlock {
+  display: flex;
+  justify-content: space-between;
+}
+
+.Profile__avatarBlock {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.Profile__personalInfo {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  margin-left: 16px;
+}
+
+.Profile__personalInfoLabel {
+  color: $text-gray;
+  line-height: 1;
+}
+
+.Profile__name {
+  @include large-text;
+  font-weight: 550;
```

Comment:
maybe set a variable or mixin as 550 so it can uniform all of the bolded font-weights